azkaban-web-server/src/main/resources/azkaban/webapp/servlet/velocity/flowexecutionpanel.vm

```diff
@@ -47,8 +47,11 @@
             </li>
             <li viewpanel="flow-parameters-panel">
               <a href="#">Flow Parameters</a>
```

Comment:
Should we make this label a bit more generic? For example: `Runtime Parameters`

Reply:
Sure, the use of "parameters" here has always confused me, as well as using "flow" in flow override (because it's not "flow level stuff", it affects all nodes). There is also mixed usage of internal fields flowOverride vs. flowParameters – but maybe it's not possible to change those – at least not those that are persisted.

Any way, could any renaming enhancements be done as a separate PR, please?

Reply:
Sure, it can be done in a separate PR.